spec/CloudCode.spec.js

```diff
@@ -2436,6 +2456,19 @@ describe('beforeLogin hook', () => {
     done();
   });
 
+  it('should trigger afterLogout hook on logout', async done => {
+    let hit = 0;
+    Parse.Cloud.afterLogout(() => {
+      hit++;
```

Comment:
After Logout does return a request object. Can you test it?

Reply:
added to tests